pkg/config/defaults.go

```diff
@@ -71,6 +71,7 @@ const (
 	DefaultStatsEnabled       = true
 	DefaultStatsAddr          = "https://stats.treeverse.io"
 	DefaultStatsFlushInterval = time.Second * 30
+	DefaultStatsFlushSize     = 1000
```

Comment:
shouldn't this be aligned with the documentation? (either setting this one to 100, or aligning the configuration's default value to 1000)